apps/sms/test/marionette/lib/views/new-message/view.js

```diff
@@ -144,10 +148,14 @@ NewMessageView.prototype = {
     return conversationView;
   },
 
-  backToInbox: function() {
+  back: function() {
```

Comment:
Not sure if it's the best name, but we have similar method pair in conversation - so kept the same principle.